Aquality.Appium.Mobile.Template/Aquality.Appium.Mobile.Template/Screens/IOS/LoginScreen.cs

```diff
@@ -1,20 +1,29 @@
-﻿using System;
-using Aquality.Appium.Mobile.Screens;
+﻿using Aquality.Appium.Mobile.Screens;
 using Aquality.Appium.Mobile.Template.Screens.Interfaces;
 using OpenQA.Selenium.Appium;
 
 namespace Aquality.Appium.Mobile.Template.Screens.IOS
 {
     public class LoginScreen : IOSScreen, ILoginScreen
     {
-        public LoginScreen() : base(MobileBy.AccessibilityId("login"), "Login")
+        public LoginScreen() : base(MobileBy.AccessibilityId("Login"), "Login")
         {
         }
 
-        public ILoginScreen SetPassword(string password) => throw new NotImplementedException("iOS screens not implemented");
+        public ILoginScreen SetUsername(string username)
+        {
+            ElementFactory.GetTextBox(MobileBy.IosNSPredicate("type == 'XCUIElementTypeTextField' AND name == 'username'"), "Username")
```

Comment:
why don't you define elemence once in constructor?

Reply:
I don't think it is a good idea to define all elements in constructor. We can use properties for elements instead of it.

Reply:
fixed